azure-nextgen-cs-appservice/AppServiceStack.cs

```diff
@@ -0,0 +1,172 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
```

Comment:
Nit (applies everywhere)
```suggestion
// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
```